app/routes/members-area/route.tsx

```diff
@@ -0,0 +1,26 @@
+import classNames from 'classnames';
+import { Outlet, NavLink } from '@remix-run/react';
+
+import styles from './route.module.scss';
+
+export default function MembersAreaPage() {
+    const menuItemClassName = ({ isActive }: { isActive: boolean }) => {
+        return classNames(styles.menuItem, { [styles.active]: isActive });
+    };
+
+    return (
+        <div className={styles.page}>
+            <div className={styles.menu}>
+                <NavLink to={'/members-area/my-account'} className={menuItemClassName}>
+                    My Account
+                </NavLink>
+                <NavLink to={'/members-area/my-orders'} className={menuItemClassName}>
+                    My Orders
+                </NavLink>
+            </div>
```

Comment:
These are tabs. Not a menu. It'd make sense to create a Tabs component or at least reusable HTML classes for building a tab bar.